project/Tests/Snapper.MSTest.Tests/MSTestSnapperTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Runtime.CompilerServices;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Snapper.Attributes;
+
+namespace Snapper.MSTest.Tests
+{
+    [TestClass]
+    public class MSTestSnapperTests
+    {
+        [TestMethod]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public void TestIfStoredSnapshotIsMatching()
+        {
+            var actual = new { TestProperty = "TestValue" };
+            actual.ShouldMatchSnapshot();

```

Comment:
Since you are just testing that the framework is detected. You can use inline snapshot which saves a snapshot being made. 

Here is an example https://github.com/theramis/Snapper/blob/master/project/Tests/Snapper.TestFrameworkSupport.Tests/XUnitFrameworkSupportTests.cs#L12